sherpa/astro/xspec/__init__.py

```diff
@@ -160,6 +163,40 @@ def set_xsxset(name, value):
         modelstrings[name] = get_xsxset(name)
 
 
+# Since these are not included in __all__, create a symbol in this
+# module to avoid "unused symbol" if they were to be imported
+# directly from _xspec.
+#
+get_xspath_manager = _xspec.get_xspath_manager
+get_xspath_model = _xspec.get_xspath_model
+
+
+def set_xspath_manager(path):
+    """Set the path to the files describing the XSPEC models.
+
+    Parameters
+    ----------
+    path : str
+        The new path.
+
+    See Also
+    --------
+    get_xspath_manager : Return the path to the files describing the XSPEC models.
+
+    Examples
+    --------
+    >>> set_xspath_manager('/data/xspec/spectral/manager')
+    """
+
+    _xspec.set_xspath_manager(path)
+    spath = get_xspath_manager()
+    if spath != path:
```

Comment:
Hmm... when I saw there were no tests for the error condition I tried to set this to the string '%$', and I got the same string from `get_xspath_manager`, so it looks like this error condition is redundant? Or is it really possible it's going to fail?

Reply:
I honestly don't know. You'd have to dig into the XSPEC model library code to see if this can happen. I'm just trying to be a bit defensive here, but maybe it's not worth it?

Reply:
I resisted the urge to remove this check we can't test.